dispatch/static/manager/package.json

```diff
@@ -54,5 +55,8 @@
   },
   "bugs": {
     "url": "https://github.com/ubyssey/dispatch/issues"
+  },
+  "dependencies": {
+    "moment": "^2.17.1"
```

Comment:
If I recall correctly, there used to be a time when saving deps as `devDependencies` was the norm for Webpack projects (since all of them are bundled anyways), but I think that's changed. I'm cool with moving everything over to `dependencies`.